src/orders/components/OrderTransaction/components/TransactionEvents/TransactionEvents.tsx

```diff
@@ -5,7 +5,7 @@ import { TransactionFakeEvent } from "@dashboard/orders/types";
 import { TableCell, TableRow } from "@material-ui/core";
 import { makeStyles, ResponsiveTable } from "@saleor/macaw-ui";
```

Comment:
suggestion: Since we're already rewriting these components, maybe we could get rid of material-ui? Especially `makeStyles` to use only one way to style components (e.g. by using `<Box>` from macaw-ui-next, or by CSS modules)